### PR TITLE
Adding atom

### PR DESCRIPTION
**Pull request by hargoniX**

### CODE REVIEW

File: static/resources.json [hunk 1]
```diff
@@ -54,6 +54,12 @@
         "url": "https://www.sublimetext.com/",
         "payment": "optional",
         "payment_description": "Nagware; will ask you to buy the full version after every X saves"
+      },
+      "Atom (Free)": {
+          "description": "A free Electron based editor, a \"hackable text editor for the 21st century\".",
```
Comment (gdude2002) [inline static/resources.json]: Should be `Electron-based`, not `Electron based`

Comment (gdude2002) [reply static/resources.json]: Perhaps also mention that Atom is maintained by the GitHub team - at least, I think that's the case, right?

**Review by gdude2002 — CHANGES_REQUESTED** [review]
Resources should be in alphabetical order

**Comment by gdude2002** [comment]
(I just noticed that VS code is at the top, so rearrange it all anyway)

File: static/resources.json (hunk 1 above)
Comment (hargoniX) [reply static/resources.json]: It is

(oh now it is alphabetic :stuck_out_tongue:)

**Pull request merged**